flixel/graphics/frames/FlxFilterFrames.hx

```diff
@@ -156,9 +156,7 @@ class FlxFilterFrames extends FlxFramesCollection
 		if (filters.length == 0)
 			return;
 		
-		while (filters.length != 0)
-			filters.pop();
-		
+		filters = filters.splice(0, filters.length);
```

Comment:
Based on the conversation in #1992, shouldn't this omit `filters =` otherwise it would be capturing the removed items?